packages/contracts/contracts/universal/KromaSoulBoundERC721.sol

```diff
@@ -106,6 +106,7 @@ abstract contract KromaSoulBoundERC721 is
         _tokenIdCounter.increment();
         _safeMint(to, tokenId);
         _setTokenURI(tokenId, uri);
+        _delegate(to, to);
```

Comment:
With this change, we may need to upgrade `SecurityCouncilToken` contract as well, right?
Accordingly I think we need to update semver of the contract to `v1.0.1`.

And this is out of scope of this PR I think. For the next time, let's make another PR.